packages/swapper/src/swappers/ChainflipSwapper/constants.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  arbitrumAssetId,
+  type AssetId,
+  btcAssetId,
+  type ChainId,
+  ethAssetId,
+  flipAssetId,
+  solAssetId,
+  usdcAssetId,
+  usdcOnArbitrumOneAssetId,
+  usdcOnSolanaAssetId,
+  usdtAssetId,
+} from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+import type { SupportedChainIds, SwapSource } from '../../types'
+import { SwapperName } from '../../types'
+
+export const CHAINFLIP_REGULAR_QUOTE = 'regular'
+export const CHAINFLIP_DCA_QUOTE = 'dca'
+export const CHAINFLIP_BAAS_COMMISSION = 5
+
+export const ChainflipSupportedChainIds = [
+  KnownChainIds.EthereumMainnet,
+  KnownChainIds.ArbitrumMainnet,
+  KnownChainIds.BitcoinMainnet,
+  KnownChainIds.SolanaMainnet,
+] as const
+
+export type ChainflipSupportedChainId = (typeof ChainflipSupportedChainIds)[number]
+
+export const ChainflipSupportedAssetIdsByChainId: Partial<Record<KnownChainIds, AssetId[]>> = {
+  [KnownChainIds.EthereumMainnet]: [ethAssetId, flipAssetId, usdcAssetId, usdtAssetId],
+  [KnownChainIds.ArbitrumMainnet]: [arbitrumAssetId, usdcOnArbitrumOneAssetId],
+  [KnownChainIds.BitcoinMainnet]: [btcAssetId],
+  [KnownChainIds.SolanaMainnet]: [solAssetId, usdcOnSolanaAssetId],
+}
+
+export const chainIdToChainflipNetwork: Partial<Record<ChainId, string>> = {
+  [KnownChainIds.EthereumMainnet]: 'eth',
+  [KnownChainIds.ArbitrumMainnet]: 'arb',
+  [KnownChainIds.BitcoinMainnet]: 'btc',
+  [KnownChainIds.SolanaMainnet]: 'sol',
+}
+
+export const CHAINFLIP_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
+  sell: ChainflipSupportedChainIds as unknown as ChainId[],
+  buy: ChainflipSupportedChainIds as unknown as ChainId[],
+}
+
+export const CHAINFLIP_SWAP_SOURCE: SwapSource = SwapperName.Chainflip
+export const CHAINFLIP_BOOST_SWAP_SOURCE: SwapSource = `${SwapperName.Chainflip} • Boost`
+export const CHAINFLIP_DCA_SWAP_SOURCE: SwapSource = `${SwapperName.Chainflip} • DCA`
+export const CHAINFLIP_DCA_BOOST_SWAP_SOURCE: SwapSource = `${SwapperName.Chainflip} • DCA • Boost`
+
+export const usdcAsset: Asset = {
```

Comment:
That's absolutely fine @CumpsD! 
Just to disambiguate here, what's the two actual flows when fees will be denominated in USDC? i.e i can see the condition is `fee.type === 'network'`and `fee.type === 'liquidity' && fee.asset === 'usdc.eth'` but not sure what this means exactly.

Asking because we may be able to simply use `buyAsset` / `sellAsset` if one of these is USDC

Reply:
Network fee is always USDC, it is the 0.10% buy & burn fee the chainflip network uses.

Liquidity is the LP fee, which I would have to double check but I believe the second leg is always USDC. It's possible this `if (fee.type === 'liquidity' && fee.asset === buyChainflipChainKey) return buyAsset` might actually never be hit.

Will double check!

Reply:
Yep, so the second leg seems to be always usdc.eth for the liquidity fee.

Reply:
Happy to keep the second branch for the sake of paranoia. Tyvm for disambiguating this! Looks like indeed, we need usdc as a static asset then, which means keeping it here as a const is absolutely fine.

Reply:
> Happy to keep the second branch for the sake of paranoia. Tyvm for disambiguating this! Looks like indeed, we need usdc as a static asset then, which means keeping it here as a const is absolutely fine.

Pretty much the reason I added all branches too :D